sa/sa.go

```diff
@@ -1717,13 +1699,9 @@ func (ssa *SQLStorageAuthority) getAllOrderAuthorizationStatuses(
 		allAuthzValidity = append(allAuthzValidity, validityInfo...)
 	}
 
-	tx, err := ssa.dbMap.Begin()
-	if err != nil {
-		return nil, err
-	}
 	for _, table := range authorizationTables {
 		var validityInfo []authzValidity
-		_, err := tx.WithContext(ctx).Select(
+		_, err := ssa.dbMap.Select(
```

Comment:
It looks like this used to be done in a transaction and now isn't. It seems like the transaction was pointless in the original code, am I correctly understanding the rationale for the change?

Reply:
Correct, the transaction was pointless in the original code. I wound up changing this while tracking down a test failure that turned out to be unrelated. I'll submit this part as a separate PR just for cleanliness.

Reply:
Up to you, I don't think its too egregious to do it in this PR.

Reply:
BTW, good catch on this - when separating into another PR, I noticed that this version was missing the `.WithContext(ctx)` call!